src/components/UriViewer.js

```diff
@@ -56,9 +57,14 @@ const isFilePreviewable = ({ size, ...metadata }) => {
   return !isBinary(metadata) && (isText(metadata) || (isImage(metadata) && size <= 1e9))
 }
 
-const isGs = uri => _.startsWith('gs://', uri)
+export const isGs = uri => _.startsWith('gs://', uri)
 
-const isDrs = uri => _.startsWith('dos://', uri) || _.startsWith('drs://', uri)
+export const isDrs = uri => _.startsWith('dos://', uri) || _.startsWith('drs://', uri)
+
+export const isAzureUri = uri => {
+  const azureRegex = RegExp('https://(.+).blob.core.windows.net')
```

Comment:
This could probably be tightened up a bit. For example, checking that this pattern appears at the start of the `uri` string. Also, I'm guessing there's a limited set of characters allowed for the storage container ID.

Reply:
I updated it to ensure that the regex starts at the beginning of the uri provided

It might be tricky to enforce any further: https://learn.microsoft.com/en-us/rest/api/storageservices/naming-and-referencing-containers--blobs--and-metadata#blob-names